snuba/migrations/snuba_migrations/profiles/0001_profiles.py

```diff
@@ -0,0 +1,88 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, DateTime, NamedTuple, String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+columns: List[Column[Modifiers]] = [
+    # primary key
+    Column("organization_id", UInt(64)),
+    Column("project_id", UInt(64)),
+    Column("transaction_id", UUID()),
+    Column("received", DateTime()),
+    # profiling data
+    Column("profile", String(Modifiers(codecs=["LZ4HC(9)"]))),
+    Column("symbols", String(Modifiers(codecs=["LZ4HC(9)"]))),
+    # filtering data
+    Column("android_api_level", UInt(32, Modifiers(nullable=True))),
+    Column("device_classification", String(Modifiers(low_cardinality=True))),
+    Column("device_locale", String(Modifiers(low_cardinality=True))),
+    Column("device_manufacturer", String(Modifiers(low_cardinality=True))),
+    Column("device_model", String(Modifiers(low_cardinality=True))),
+    Column("device_os_build_number", String(Modifiers(low_cardinality=True))),
+    Column("device_os_name", String(Modifiers(low_cardinality=True))),
+    Column("device_os_version", String(Modifiers(low_cardinality=True))),
+    Column("duration_ns", UInt(64)),
+    Column("environment", String(Modifiers(nullable=True, low_cardinality=True))),
+    Column("error_code", String(Modifiers(low_cardinality=True, nullable=True))),
+    Column("error_description", String(Modifiers(low_cardinality=True, nullable=True))),
```

Comment:
The columns which have `low_cardinality` set to True, are these user input strings or machine generated strings? Just wanted to check. You might not want to have columns which are user input strings

Reply:
I'd remove it since it's actually optional for `ReplacingMergeTree` but `mypy` forces me to have something as `version_column`.

```
snuba/migrations/snuba_migrations/profiles/0001_profiles.py:50: error: Missing positional argument "version_column" in call to "ReplacingMergeTree"
```

Reply:
You can make the `version` parameter optional in https://github.com/getsentry/snuba/blob/master/snuba/migrations/table_engines.py#L96. Clickhouse `ReplacingMergeTree` states that it is optional https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/replacingmergetree/